deploy/crds/kieapp.crd.yaml

```diff
@@ -272,6 +272,18 @@ spec:
                     description: The password to use for keystore generation.
                     format: password
                     type: string
+                  secretAdminCredentialsReference:
+                    description: SecretAdminReference is a reference to the secret
+                      containing this user's password
+                    format: password
```

Comment:
is this format really needed for setting up string property of some secret name?

Reply:
No